infra/terraform/modules/user_nfs_softnas/ec2.tf

```diff
@@ -0,0 +1,142 @@
+data "aws_ami" "softnas" {
+  most_recent = true
```

Comment:
We wouldn't redeploy SoftNAS to update it, as it handles software updates itself via the GUI - the `most_recent` flag here is just to ensure that you get the most up-to-date version when you deploy it for the first time.